app.js

```diff
@@ -1,23 +1,33 @@
-const db = require('./db/connection.js')
-const express = require('express')
-const {getAllTopics, getApis, wrongPath} = require('./controller/app.controller.js')
+const db = require("./db/connection.js");
+const express = require("express");
+const {
+    getAllTopics,
+    getApis,
+    wrongPath,
+    getArticleById,
+} = require("./controller/app.controller.js");
 
-const app = express()
+const app = express();
 
-app.use(express.json())
+app.use(express.json());
 
-app.get('/api/topics', getAllTopics)
+app.get("/api/topics", getAllTopics);
 
-app.get('/api', getApis)
+app.get("/api/articles/:id", getArticleById);
+
+app.get("/api", getApis);
+
+
+
+
+app.get("*", wrongPath);
 
-app.get('*', wrongPath);
 
 
 app.use((err, request, response, next) => {
```

Comment:
We also now want some PSQL error handling to check the PSQL code and throw the appropriate error. For instance Status 400, invalid ID, e.g. string of "not-an-id"